.circleci/config.yml

```diff
@@ -42,5 +46,6 @@ workflows:
 
   my_workflows:
     jobs:
-      - create_deb_package
-      - end_to_end_test
+      - create_deb_package:
+          context: ooni
+      #- end_to_end_test
```

Comment:
Did you comment out the end_to_end test because it wasn't working?

I had similar issues getting it to work inside of: https://github.com/ooni/api/pull/240